trackpy/feature.py

```diff
@@ -457,30 +459,39 @@ def locate(raw_image, diameter, minmass=100., maxsize=None, separation=None,
 
     Parameters
     ----------
-    image : image array (any dimensions)
-    diameter : feature size in px
+    raw_image : array
```

Comment:
re api consistency at day job, should this stay as `image`.  The description should be something like 'the image to find features in'